husky_directory/static/css/structure.css

```diff
@@ -1,5 +1,17 @@
 /* Not in our normal CSS; custom */
 
+.table-striped > tbody > tr:nth-child(odd) {
```

Comment:
Looks good on my screen. If this ain't it, then feel free to give it your best shot 👍